packages/documentation/src/pages/platform/tools/downtime-notifications.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Va.gov downtime notifications
+tags: pagerduty,maintenance windows,scheduled downtime
+---
+
+# Downtime Notifications
+Downtime Notications are a mechanism for handling outages of external services in the consuming front-end applications. 
+
+## Overview
+- Available as a React component in [`vets-website`](https://github.com/department-of-veterans-affairs/vets-website/blob/cb1d8072a94b8847e0af0c697ff879b42db354f7/src/platform/monitoring/DowntimeNotification/index.jsx).
```

Comment:
Link text surrounded by back ticks looks kinda weird on the site, I've been leaving out the back ticks if it's a link.